commondata/ATLAS_ggF_13TeV_2015.yaml

```diff
@@ -1,4 +1,11 @@
-dataset_name: ATLAS_H_13TeV_2015_pTH
+dataset_name: ATLAS_ggF_13TeV_2015
```

Comment:
does this mean that the dataset was mismatched?

Reply:
The numbers in this yaml file correspond to Fig. 2a in [ATLAS_ggF_13TeV_2015](https://arxiv.org/pdf/1805.10197.pdf), so it was not labelled correctly before. The smefit2.0 paper also writes [ATLAS_H_13TeV_2015_pTH](http://cdsweb.cern.ch/record/2682844/files/ATLAS-CONF-2019-032.pdf) in Table 5.2, but this should be ATLAS_ggF_13TeV_2015 if you look for instance at the number of datapoints. We never included ATLAS_H_13TeV_2015_pTH in smefit2.0.